### PR TITLE
Twitter display implementation dialogue background

### PR DESCRIPTION
**Pull request by andrescalandriglobant**
Added the chatboxes background to the twitter display for both left and right tweets

### CODE REVIEW

File: js/displays/et.displaywall.display.js [hunk 1]
```diff
@@ -138,6 +138,55 @@ var Display = (function () {
...
+    };
+    var duration = self.opts.config.duration;
+
+    var s;
+    new TWEEN.Tween(from)
+      .to({
+        opacity: toOpacity
+      })
+      .onUpdate(function() {
+        o = this.opacity;
```
Comment (felipe-sanchez) [inline js/displays/et.displaywall.display.js]: if we add self, maybe we should use it ;) . Mmm, maybe just this.mesh.material.opacity = self.opacity without the "o".

File: js/displays/et.displaywall.display.js [hunk 2]
```diff
@@ -138,6 +138,55 @@ var Display = (function () {
...
+    };
+    var duration = self.opts.config.duration;
+
+    var s;
+    new TWEEN.Tween(from)
+      .to({
+        opacity: 0.001
+      })
+      .onUpdate(function() {
+        o = this.opacity;
```
Comment (felipe-sanchez) [inline js/displays/et.displaywall.display.js]: same as above.

File: js/displays/et.displaywall.twitter.js [hunk 3]
```diff
@@ -16,6 +16,8 @@ var YarvisTwitter = (function () {
       var self = this;
       var divId = 'twitter-content';
       var divIdRight = 'twitter-content-right';
+      var container;
```
Comment (felipe-sanchez) [inline js/displays/et.displaywall.twitter.js]: lets use a more descriptive name here. tweetsContainer comes to mind.

File: js/displays/et.displaywall.twitter.js [hunk 4]
```diff
@@ -303,6 +309,47 @@ var YarvisTwitter = (function () {
...
+      chatbox.position.y =  0.33 * self.base.yarvisOpts.YARVIS_HEIGHT;
+      self.base.mesh.add(chatbox);
+
+      //Animation
+      self.base.fadeMeshIn(chatbox, 1.0, 250);
+      self.base.fadeMeshOut(chatbox, 1.0, 1000);
+
+      //We add the right Chatbox
+      texture = THREE.ImageUtils.loadTexture("/assets/images/twitter/conversation_box_right.png");      
+      geometry = new THREE.PlaneGeometry(self.base.yarvisOpts.YARVIS_WIDTH * 0.1557,self.base.yarvisOpts.YARVIS_HEIGHT * 0.5074);
```
Comment (felipe-sanchez) [inline js/displays/et.displaywall.twitter.js]: let's split lines longer than 80 into two lines with the second one being indented.

**Comment by ncglobant** [comment]
I am ok with these changes.

**Comment by felipe-sanchez** [comment]
LGTM

**Pull request merged**